### PR TITLE
Show submit bug and juju card on entity details pages

### PR DESCRIPTION
**Pull request by Lukewh**

### CODE REVIEW

**Comment by Lukewh** [comment]
## QA
Go to some charms and bundles and see that the 'Submit Bug' link in the header is there and the generated 'Juju Card' is in the right hand column (at the bottom).
*If the 'Submit Bug' link is missing, verify it's not a bug by comparing with the charm/ bundle on https://jujucharms.com/store*

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7131/

**Comment by frankban** [comment]
Fixes https://github.com/juju/juju-gui/issues/2726

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7136/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7137/

File: jujugui/static/gui/src/app/components/entity-details/content/test-content.js [hunk 1]
```diff
@@ -31,8 +31,22 @@ function _generateTagItem(tag, fn) {
...
 
+function generateScript(isBundle) {
+  let id = 'trusty/django-123';
+  if (isBundle) {
+    id = 'django-cluster';
+  }
+
+  return '<script ' +
+  'src="https://assets.ubuntu.com/v1/juju-cards-v1.5.0.js"></script>\n' +
+  '<div class="juju-card" data-id="'+id+'"></div>';
```
Comment (Lukewh) [inline jujugui/static/gui/src/app/components/entity-details/content/test-content.js]: I initially used template literals, but linting complained about long lines. So then you add newlines, and the newlines are included in the output.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7138/

**Comment by Lukewh** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**